src/errors/index.js

```diff
@@ -0,0 +1,38 @@
+import React from "react";
+import { render } from "react-dom";
+import { Provider } from "react-redux";
+import { compose, createStore, applyMiddleware } from "redux";
+import thunk from "redux-thunk";
+
+import reducers from "./reducers";
+import { ERROR } from "./actions";
+import ErrorView from "./error-view";
+
+export default class Errors {
+    constructor(el) {
+        this.el = el;
+
+        const store = compose(
+            applyMiddleware(thunk)
+        )(createStore)(reducers);
+
+        this.store = store;
+
+        render(
+            <Provider store={this.store}>
+                <ErrorView />
+            </Provider>,
+            this.el
+        );
+    }
+
+    render(err) {
+        let { dispatch } = this.store;
+        var info = err.info;
+        var error = (info && info.err && info.err.message) || err.message;
```

Comment:
it would be nice to print out a title using the `code` attribute that outrigger api returns in all errors. I think this would help users distinguish between a compilation error, some outrigger api error, or some random js error that we caused.